thoth/storages/graph/models.py

```diff
@@ -1510,6 +1530,7 @@ class PythonPackageMetadataDistutils(Base, BaseExtension):
         RPMRequirement,
         SoftwareEnvironment,
         VersionedSymbol,
+        KebechetGithubAppInstalltions
```

Comment:
@pacospace Should I add it here?

Reply:
yes it is a main model, but the list is in alphabetical order :)

Reply:
Thanks!